### DISCUSSION TITLE
Roadmap brainstorm

### DISCUSSION THREAD

**Discussion by jsmorabito** [athensresearch/athens#715] [General]
Inspired by #703 and many questions about what's on the roadmap. Up till now the "roadmap" has been relatively informal and going forward I think it would be useful to have a central place that users and prospective contributors can look to for our future plans. Does anyone have any suggestions or know of any other projects we could emulate?

First question to ask is, what is the purpose of a roadmap for Athens, of which I see 3 currently:
1. Show users what is being worked on or what features they can expect in the future (at a high level)
2. Provide entry points for new, potential contributors by showing them whats being worked on and what needs to be worked on
3. Future authoring of the project, roadmap could give us a space to see and discuss the priority of particular features or directions of the project

Three different styles that we might consider:
1. [Githubs public roadmap](https://github.com/github/roadmap/projects/1)
   - divided into quarters with a future category at the end

2. [Obsidians public roadmap](https://trello.com/b/Psqfqp7I/obsidian-roadmap)
   - divided into sections: long term, short term, working on, done (available in next release), shipped

3. [Gitlabs roadmap](https://about.gitlab.com/direction/maturity/)
   - seems promising, wondering how we could adapt it to our project...

Variable(s) to consider:
- high volatility presented by nature of being a start up and new tech/frontier means that it should have a flexible structure and focus on bigger picture issues/features (Filter, Sort, Latex support, not "Help button in top nav" *made this up) 
-
  
  Because of the high volatility mentioned I recommend a roadmap that's not tied to strict dates, similar to obsidians, because some features or initiatives may be taken up by contributors working with or apart from the core team which would make setting expectations unrealistic. What are your thoughts?

**Comment by josephjacks** [comment]
As a new power-user (Mac only for now), I'd love to be able to submit feature requests from the app itself and even if I don't want to do that, being able to click from the app to a central place would be great! I'm a fan of how GitHub manages their roadmap.

**Reply by jsmorabito** [reply]
thats a cool idea! the feature request process could definitely be upgraded and streamlined, and could probably use a discussion topic all on its own. I started thinking briefly about it here on this document, feel free to leave comments https://docs.google.com/document/d/1mM1hA3nuLSWy29XOUhBTDRJN7NtQxoaT0UYZKA4aQuo/edit#heading=h.cxmw4a5hc4q2

**Reply by domwass** [reply]
As I was the one who asked for the roadmap, I just wanted to say that I wanted to see where the project is going. I would not need an elaborated list with proposed dates, but rather an outline with envisaged milestones. So more in the sense of Obsidian’s roadmap. I think that would give interested users the possibility to consider where the project is heading.

As an aside: I have now managed to get org-roam running on my Emacs, and as I already use org-mode for my daily todos etc. and the graph visualization is working well with org-mode-server, I will probably stick with that for the moment. But I think that Athens is really good already because of its clean and simple design. I will keep an eye on it and I’m curious where it’s heading.

**Comment by jefftangx** [comment]
GitHub's public roadmap: https://github.com/github/roadmap/projects/1

**Comment by jsmorabito** [comment]
v1 roadmap here: https://github.com/athensresearch/athens/projects/12
